.github/workflows/integration.yaml

```diff
@@ -261,6 +261,46 @@ jobs:
             test -p ibc-integration-test --features forward-packet --no-fail-fast -- \
             --nocapture --test-threads=1 forward::
 
+  ics31:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        chain:
```

Comment:
I think you need to add the new `stride-no-admin` package here.

Reply:
Testing requires to have one Stride chain and one Gaia chain, if the `stride-no-admin` is added to the `matrix` wouldn't result in having one test running 2 Gaia chains and another test running two Stride chains ?
The idea here was to have the possibility to test multiple Gaia versions if we want/need to, by simply adding them to the `matrix`. Resulting in every test being:

* `Chain from matrix` <-> `Stride chain`

What do you think ?

Reply:
Hmm, I think if you want a CI matrix, then putting stride in it also result in all the combinations with stride executed.